bundles/org.openhab.binding.fronius/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -120,6 +123,46 @@
 		</config-description>
 	</thing-type>
 
+	<!-- Ohmpilot Thing Type -->
+	<thing-type id="ohmpilot">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bridge"/>
+		</supported-bridge-type-refs>
+		<label>Fronius Ohmpilot</label>
+		<description>Fronius Ohmpilot</description>
+
+		<channels>
+			<channel id="ohmpilotpowerrealsum" typeId="ohmpilot_powerreal">
```

Comment:
Why do you prefix all your channels with "ohmpilot" ? This looks redundant while they are channels for a thing type "ohmpilot".
Look at the channels for the "meter" thing type, they are not prefixed.